.gitignore

```diff
@@ -3,6 +3,10 @@ installs
 *.log
 node_modules/
 
+# Added for pedestal framework test
```

Comment:
Instead of adding this to the global `.gitignore` for the entire project, it would be better to create a file at `frameworks/Clojure/pedestal/.gitignore` and add these rules there